docs/labs/index.rst

```diff
@@ -18,9 +18,9 @@ If we get lots of positive feedback about any one of these projects then we will
 Flocker Docker plugin with CLI and GUI
 ======================================
 
-By way of example, here is a 55 second demo of the :ref:`Flocker Docker plugin <labs-docker-plugin>` provisioning portable Flocker volumes and moving them between hosts directly from the Docker CLI.
+By way of example, here is a 55 second demo of the :ref:`Flocker Docker plugin <docker-plugin>` provisioning portable Flocker volumes and moving them between hosts directly from the Docker CLI.
```

Comment:
I too prefer the latter, but given that these docs are still here, I'd recommend updating all references to be consistent. Labs projects can get away with different coding structures and standards than we use in engineering, but the way I look at it, their docs are still part of our docs and should be held to the same standard of consistency. The labs plugin had a different name "Flocker Docker Plugin" for labs, but it's not a labs project anymore, it's an official product being referenced by labs docs, so I think it should be referenced with the correct name.